src/coreclr/interpreter/compiler.cpp

```diff
@@ -52,6 +52,12 @@ void DECLSPEC_NORETURN Interp_NOMEM()
     throw std::bad_alloc();
 }
 
+void DECLSPEC_NORETURN Interp_CompilationFail(const char *msg)
```

Comment:
Nit: Should we call it `NO_WAY` like in a regular JIT?

I think the more similar the interpreter JIT to the regular JIT, the better.